roles/contiv/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: reload systemd
+  command: systemctl --system daemon-reload
+
+- name: restart netmaster
+  service: name=netmaster state=restarted
+  when: netmaster_started.changed == false
+
+- name: restart netplugin
+  service: name=netplugin state=restarted
+  when: netplugin_started.changed == false
```

Comment:
just cosmetic but will be nice to be consistent with the format - i prefer
```
- name: restart netplugin
  service: 
    name: netplugin 
    state: restarted
  when: netplugin_started.changed == false

```
but regardless just pick one and be consistent all the way through, currently is a mix ;)